aspnetcore/mvc/controllers/filters.md

```diff
@@ -14,68 +14,64 @@ uid: mvc/controllers/filters
+Filters run within the *MVC action invocation pipeline*, sometimes referred to as the *filter pipeline*.  The filter pipeline runs after MVC selects the action to execute.
 
 ![The request is processed through Other Middleware, Routing Middleware, Action Selection, and the MVC Action Invocation Pipeline. The request processing continues back through Action Selection, Routing Middleware, and various Other Middleware before becoming a response sent to the client.](filters/_static/filter-pipeline-1.png)
 
-Different filter types run at different points within the pipeline. Some filters, like authorization filters, only run before the next stage in the pipeline, and take no action afterward. Other filters, like action filters, can execute both before and after other parts of the pipeline execute, as shown below.
+### Filter Types
 
-![The request is processed through Authorization Filters, Resource Filters, Model Binding, Action Filters, Action Execution and Action Result Conversion, Exception Filters, Result Filters, and Result Execution. On the way out, the request is only processed by Result Filters and Resource Filters before becoming a response sent to the client.](filters/_static/filter-pipeline-2.png)
+Each filter type is executed at a different stage in the filter pipeline.
+
+* [Authorization filters](#authorization-filters) run first and are used to determine whether the current user is authorized for the current request. They can short-circuit the pipeline if a request is unauthorized. 
 
-### Selecting a Filter
+* [Resource filters](#resource-filters) are the first to handle a request after authorization.  They can run code before the rest of the filter pipeline, and after the rest of the pipeline has completed. They're useful to implement caching or otherwise short-circuit the filter pipeline for performance reasons.
 
-[Authorization filters](xref:mvc/controllers/filters#authorization-filters) are used to determine whether the current user is authorized for the request being made.
+* [Action filters](#action-filters) can run code immediately before and after an individual action method is called. They can be used to manipulate the arguments passed into an action and the result returned from the action.
 
-[Resource filters](xref:mvc/controllers/filters#resource-filters) are the first filter to handle a request after authorization, and the last one to touch the request as it is leaving the filter pipeline. They're especially useful to implement caching or otherwise short-circuit the filter pipeline for performance reasons.
+* [Exception filters](#exception-filters) come after Action filters in the pipeline and are used to apply global policies to unhandled exceptions that have occurred up to that point. 
```

Comment:
@rynowak 

>Exception filters **surround** action filters and model binding.

By "surround" do you mean Exception filters can run after each stage? From the diagram, it appears that if Model Binding throws, it would be too late to let Action filters and methods run as if the exception was handled, because Exception filters only run after Action filters.  If that's incorrect, we could either modify the diagram or explain this in the section dedicated to Exception filters later in the article. Otherwise what do you mean by "surround"?

Also, you don't mention Resource filters, but the diagram implies that an Exception filter would run for an exception thrown by OnResourceExecuting -- is that not true?

Given the complexity, would this be close enough for the summary, leaving details for the Exception Filters section later in the article:

>[Exception filters](#exception-filters) are used to apply global policies to unhandled exceptions that occur before anything has been written to the response body.